### PR TITLE
Patch out country-specific presets

### LINKED ISSUE #4916: No label / feature for post boxes
(works with iD presets 5.2.1)
Post boxes don't have a label (text below the question) any more, because the feature search returns nothing. This matters for the check existence quest, which requires a feature to be asked.

Search works when either using iD presets v5.2.1, or when searching by country (though results may obviously depend on country). Seems in iD presets 6.0.0 it's assumed post boxes are only a local thing for some reason...

Anyway, similar things might be happening for other features, just no one noticed. Maybe feature search should always specify a country?

### PR DESCRIPTION
**Pull request by westnordost**
* Fixes #4916
* Replaces and closes #5969

### CODE REVIEW

**Comment by westnordost** [comment]
Note for reviewers:

See https://raw.githubusercontent.com/openstreetmap/id-tagging-schema/main/dist/presets.json and search for "`locationSet`" to see what is removed/patched.

File: app/src/androidTest/java/de/westnordost/streetcomplete/util/NameAndLocationLabelTest.kt [hunk 1]
```diff
@@ -45,8 +45,6 @@ class NameAndLocationLabelTest {
         )))
     }
 
-    // https://github.com/streetcomplete/StreetComplete/issues/2640
-    @Ignore("https://github.com/streetcomplete/StreetComplete/issues/4916")
```
Comment (FloEdelmann) [inline app/src/androidTest/java/de/westnordost/streetcomplete/util/NameAndLocationLabelTest.kt:49]: I think the first comment should stay, to give some context what exactly is tested here (similar to the other test cases). But the `@Ignore(…)` can of course be deleted.

**Review by FloEdelmann — APPROVED** [review]
This looks much cleaner than #5969 :+1::slightly_smiling_face:

**Comment by westnordost** [comment]
So, the following is patched **out**:
- maxspeed sign had a different icon in US, CA, LR
- `office=bail_bond_agent` was only available in US, PH
- `historic=memorial` + `memorial=stolperstein` was only available in Europe
- `amenity=post_box` had the field `drive_through` more prominently displayed in the US
- `amenity=townhall` + `townhall:type=barangay` was only available in PH
- `highway=toll_gantry` was not available in DE
- `network:type=node_network` vertices were only available in BE, DE, LU, NL
- `man_made=footwear_decontamination` was only available in AU, NZ (*come on! As if it wasn't possible to have footwear decontamination facilities elsewhere! This is one good example why this whole thing reeks of shoehornification!*)
- trunk, motorway and primary road links showed advisory maxspeed more prominently than maxspeed, for some reason, in US, CA
- "moped link roads" were only available in NL

Many of these are not relevant for StreetComplete, as preset fields are not used.

**Pull request merged**